### ISSUE TITLE
Feature Request: Ability to "item level to char" but set minimum level to start search for

### ISSUE THREAD

**Issue by ensiform** [apocalyptech/bl3-cli-saveedit#21]
Example, if you have low level items you wish to keep for gameplay reasons but want to bump other items.

Say you have a Level 1 Brawler Ward you want to keep but you want to bump 57+ to char level.

**Comment by apocalyptech** [comment]
Yeah, seems like that'd be easy enough to put in!  I've got a busy week coming up; might be able to get to it relatively quickly but it could be into next week.
